test/Yoti.Auth.Tests/ShareUrl/Extensions/LocationConstraintExtensionBuilderTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Yoti.Auth.ShareUrl.Extensions;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+
+namespace Yoti.Auth.Tests.ShareUrl.Extensions
+{
+    [TestClass]
+    public class LocationConstraintExtensionBuilderTests
+    {
+        private const double _someLatitude = 1d;
+        private const double _someLongitude = 2d;
+        private const double _someRadius = 3d;
+        private const double _someUncertainty = 4d;
+
+        [DataTestMethod]
+        [DataRow(-91)]
+        [DataRow(91)]
+        [TestMethod]
+        public void ShouldFailForLatitudesOutsideOfRange(double latitude)
+        {
+            Assert.ThrowsException<ArgumentException>(() =>
+            {
+                new LocationConstraintExtensionBuilder()
+                .WithLatitude(latitude)
+                .Build();
+            });
+        }
+
+        [DataTestMethod]
+        [DataRow(-181)]
+        [DataRow(181)]
+        [TestMethod]
+        public void ShouldFailForLongitudesOutsideOfRange(double longitude)
+        {
+            Assert.ThrowsException<ArgumentException>(() =>
```

Comment:
Could this be `ArgumentOutOfRangeException`?

Reply:
Updated in 70924a6